debug/src/debug.js

```diff
@@ -221,34 +227,33 @@ export function initDebug() {
 
 		if (vnode._component && vnode._component.__hooks) {
 			let hooks = vnode._component.__hooks;
-			(hooks._list || []).forEach(hook => {
-				if (hook._callback && (!hook._args || !Array.isArray(hook._args))) {
-					/* istanbul ignore next */
-					console.warn(
-						`In ${vnode.type.name || vnode.type} you are calling useMemo/useCallback without passing arguments.\n` +
-						`This is a noop since it will not be able to memoize, it will execute it every render.`
-					);
-				}
-			});
+			if (hooks._list && Array.isArray(hooks._list)) {
```

Comment:
Wondering if we could just do `Array.isArray(hooks._list)`. In theorie it should bail out for falsy cases too.

Reply:
Good cal! Will update